sdk/python/feast/data_source.py

```diff
@@ -43,10 +43,14 @@ class FileOptions:
     """
 
     def __init__(
-        self, file_format: Optional[FileFormat], file_url: Optional[str],
+        self,
+        file_format: Optional[FileFormat],
+        file_url: Optional[str],
+        s3_endpoint_override: Optional[str],
```

Comment:
Can we please add a Google style docstring for arguments here?

Reply:
fixed